config/initializers/folio_client.rb

```diff
@@ -1,14 +1,24 @@
 # frozen_string_literal: true
 
-if !Rails.env.test?
-  FolioClient.configure(
-    url: Settings.folio.url,
-    login_params: {
-      username: Settings.folio.username, password: Settings.folio.password
-    },
-    okapi_headers: {
-      "X-Okapi-Tenant": Settings.folio.tenant_id,
-      "User-Agent": "folio_client #{FolioClient::VERSION}; dataloading-management #{Rails.env}"
-    }
-  )
+# Configure folio_client singleton
+begin
+  if !Rails.env.test?
```

Comment:
If we're going to wrap client configuration in a `begin` block, I think we can do away with the `Rails.env` check.